musicdroid/src/main/java/org/catrobat/musicdroid/pocketmusic/projectselection/dialog/AboutDialog.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Musicdroid: An on-device music generator for Android
+ * Copyright (C) 2010-2015 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.musicdroid.pocketmusic.projectselection.dialog;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+
+import org.catrobat.musicdroid.pocketmusic.R;
+
+public class AboutDialog extends DialogFragment {
+
+    public static AboutDialog newInstance(int title) {
+        AboutDialog frag = new AboutDialog();
+        Bundle args = new Bundle();
+        frag.setArguments(args);
+        return frag;
+    }
+
+    @Override
+    public Dialog onCreateDialog(Bundle bundle) {
+
+        View view = LayoutInflater.from(getActivity()).inflate(R.layout.dialog_about, null);
+        Dialog aboutDialog = new AlertDialog.Builder(getActivity())
+                .setView(view)
+                .setTitle(R.string.app_name)
+                .setNeutralButton(R.string.ok, new DialogInterface.OnClickListener() {
```

Comment:
if you use "android.R.string.ok" there is no need for changing the strings.xml file